GitUI/UserControls/FileStatusList.cs

```diff
@@ -493,6 +513,81 @@ int GetLastIndex()
             }
         }
 
+        private void FileStatusListView_DrawSubItem(object sender, DrawListViewSubItemEventArgs e)
```

Comment:
please move it below to where the other event handlers are.
bonus if you observe the A-Z order

Reply:
ok